ci-operator/step-registry/ipi/conf/aws/route53/ipi-conf-aws-route53-ref.yaml

```diff
@@ -0,0 +1,17 @@
+ref:
+  as: ipi-conf-aws-route53
+  from_image:
+    namespace: ocp
+    name: "4.10"
+    tag: upi-installer
+  grace_period: 10m
+  commands: ipi-conf-aws-route53-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: COMPUTE_NODE_TYPE
```

Comment:
sounds like we do not need this env var in this case

Reply:
Removed it from the file.